src/main/java/org/mockito/Mockito.java

```diff
@@ -68,36 +68,25 @@
  *      <a href="#35">35. (new) Custom verification failure message (Since 2.1.0)</a><br/>
  *      <a href="#36">36. (new) Java 8 Lambda Matcher Support (Since 2.1.0)</a><br/>
  *      <a href="#37">37. (new) Java 8 Custom Answer Support (Since 2.1.0)</a><br/>
- *
  * </b>
  *
- * <h3 id="0">0. <a class="meaningful_link" href="#verification">Migrating to 2.1.0</a></h3>
+ * <h3 id="0">0. <a class="meaningful_link" href="#verification">Migrating to Mockito 2</a></h3>
```

Comment:
The href is incorrect and duplicated with [line 84](https://github.com/mockito/mockito/pull/628/files#diff-ddbc0d47e06a589778a6efe24c5d71d3R84)